backend/app/Models/User.php

```diff
@@ -37,4 +37,36 @@ public function getToken()
     {
         return auth()->login($this);
     }
+
+    /**
+     * Return boolean, whether or not user is following user_id.
+     *
+     * @return bool
+     */
+    public function isFollowing($user_id)
+    {
+        return Follow::where([
```

Comment:
This is quicker with eloquent: `$isFollowing = $this->following->contains($user_id);`

You will want to define two relationships on this User model: `public function followers()` and `public function following()`, which should return a `return $this->belongsToMany('App\User', 'follows', 'user_id', 'follower_id');`.

NOTE: you should swap `user_id` and `follower_id` in the return to get the correct results for `followers()` and `following()`